command/state_mv.go

```diff
@@ -36,7 +36,8 @@ func (c *StateMvCommand) Run(args []string) int {
 	cmdFlags.StringVar(&c.statePath, "state", "", "path")
 	cmdFlags.StringVar(&statePathOut, "state-out", "", "path")
 	if err := cmdFlags.Parse(args); err != nil {
-		return cli.RunResultHelp
```

Comment:
I do wonder what RunResultHelp was (and if it is useful, or if it remains useful?)

Reply:
Ah yeah, it took me awhile to track this down.  

This is an exit code from `"github.com/mitchellh/cli"`. When it's used as an exit code, the cli package prints the usage text before exiting. It's used in other tf commands, but in this case (because `cmdFlags.Parse` also prints the usage) it resulted in the help/usage text getting printed twice. 

I doubly appreciate the question because when I went back to check something I found the last bit of ~magic wherein the flags package was printing the usage, thanks!  👏

Reply:
thanks for the info, and glad it helped! 🙌